[2022-03-01 01:49:51,487] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: example_bash_operator.run_after_loop 2022-02-27T00:00:00+00:00 [queued]>
[2022-03-01 01:49:51,509] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: example_bash_operator.run_after_loop 2022-02-27T00:00:00+00:00 [queued]>
[2022-03-01 01:49:51,510] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2022-03-01 01:49:51,510] {taskinstance.py:1043} INFO - Starting attempt 1 of 1
[2022-03-01 01:49:51,511] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2022-03-01 01:49:51,520] {taskinstance.py:1063} INFO - Executing <Task(BashOperator): run_after_loop> on 2022-02-27T00:00:00+00:00
[2022-03-01 01:49:51,524] {standard_task_runner.py:52} INFO - Started process 70 to run task
[2022-03-01 01:49:51,531] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'example_bash_operator', 'run_after_loop', '2022-02-27T00:00:00+00:00', '--job-id', '11', '--pool', 'default_pool', '--raw', '--subdir', '/home/airflow/.local/lib/python3.6/site-packages/airflow/example_dags/example_bash_operator.py', '--cfg-path', '/tmp/tmpk4d_oevo', '--error-file', '/tmp/tmpnwccir0y']
[2022-03-01 01:49:51,532] {standard_task_runner.py:77} INFO - Job 11: Subtask run_after_loop
[2022-03-01 01:49:51,600] {logging_mixin.py:104} INFO - Running <TaskInstance: example_bash_operator.run_after_loop 2022-02-27T00:00:00+00:00 [running]> on host aaa141cd3a10
[2022-03-01 01:49:51,656] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=example_bash_operator
AIRFLOW_CTX_TASK_ID=run_after_loop
AIRFLOW_CTX_EXECUTION_DATE=2022-02-27T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-02-27T00:00:00+00:00
[2022-03-01 01:49:51,659] {bash.py:135} INFO - Tmp dir root location: 
 /tmp
[2022-03-01 01:49:51,661] {bash.py:158} INFO - Running command: echo 1
[2022-03-01 01:49:51,671] {bash.py:169} INFO - Output:
[2022-03-01 01:49:51,673] {bash.py:173} INFO - 1
[2022-03-01 01:49:51,674] {bash.py:177} INFO - Command exited with return code 0
[2022-03-01 01:49:51,709] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=example_bash_operator, task_id=run_after_loop, execution_date=20220227T000000, start_date=20220301T014951, end_date=20220301T014951
[2022-03-01 01:49:51,742] {taskinstance.py:1220} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2022-03-01 01:49:51,786] {local_task_job.py:146} INFO - Task exited with return code 0
